openy_location/modules/openy_loc_branch/config/optional/field.field.node.branch.field_location_directions.yml

```diff
@@ -0,0 +1,29 @@
+langcode: en
```

Comment:
this should be in original place https://github.com/open-y-subprojects/openy_features/blob/main/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_directions.yml